ui/src/combobox.tsx

```diff
@@ -55,32 +57,60 @@ export interface Combobox {
   trigger?: B
 }
 
+let newId = 1
```

Comment:
There were some issue with `newId++` on `useState`. Even though the value is only used for the first render phase, `newId` was still being incremented. I changed to a new approach with `useRef`.

Reply:
The question is why do we need artificial num incremental ids instead of textual values of options? I think it's safe to assume all the options will be unique + using incremental ids will cause problems if the user decides to update combobox options from Python - React will not know how to update correctly since keys will be the same (incremental numerics).

Reply:
Ok, I will set the keys to textual values.